### DISCUSSION TITLE
Welcome to nntrainer Discussions!

### DISCUSSION THREAD

**Discussion by myungjoo** [nnstreamer/nntrainer#1931] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kibeomi** [comment]
안녕하세요 nntrainer를 이용해보고 싶은 학생인데
nntrainer를 android studio에서 사용해보고 싶어 폴더를 오픈해도 run 버튼이 안눌려요
![image](https://user-images.githubusercontent.com/92261412/227419538-d730383b-92e6-440e-85eb-74cc1b350ab8.png)
ndk랑 여러 해결방법 찾아서 해보는데 잘안됩니다.

**Reply by myungjoo** [reply]
nntrainer는 application이 아닌 library입니다. nntrainer의 API를 사용해서 Android Application을 작성하셔야 Android Studio에서 실행하실 수 있겠지요?

**Reply by jijoongmoon** [reply]
https://github.com/nnstreamer/nntrainer/tree/main/Applications/Android 

이곳에 있는 Source들을 참고해서 보시면 될것 같습니다.  해당 Directory에는 kotlin  version 과 Java Version으로 되어 있고, 각각 Resnet18을 Android에서 동작 시킬수 있는 Application 형태로 구성되어 있습니다.

그리고 README.md 에 어떻게 Compile 하는지 적혀 있으니 한번 확인 해보세요.
